kong/plugins/statsd/schema.lua

```diff
@@ -11,6 +11,7 @@ local metrics = {
 
 return {
   fields = {
+    prefix = {type = "string", default = "kong."},
```

Comment:
The `default` field applies only to new elements, not existing ones. But that's not very clear (we could probably improve the documentation for this).